portal-web/test/functional/com/liferay/portalweb/macros/FormPortletConfiguration.macro

```diff
@@ -50,9 +50,15 @@ definition {
 
 		FormPortletNavigator.gotoSelectForm();
 
+		if (isSet(formTitle)) {
+			var formName = ${formTitle};
+		}
+
 		FormPortletConfiguration.selectForm(formTitle = ${formTitle});
 
 		PortletEntry.save();
+
+		FormPortletNavigator.exitWidgetModal();
```

Comment:
It used to be used 'Navigator.gotoPage(pageName = "${pageNameHere});' after select a form, where simply closing the modal was enough